reference/docs-conceptual/dev-cross-plat/create-help-using-platyps.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Create XML-based help using PlatyPS
+description: Using PlatyPS is the fast and efficient way to create XML-based help.
+ms.date: 07/21/2020
+---
+# Create XML-based help using PlatyPS
+
+When creating a PowerShell module, it is always recommended that you include detailed help for the
+cmdlets you create. If your cmdlets are implemented in compiled code, you must use the XML-based
+help. This XML format is known as the Microsoft Assistance Markup Language (MAML).
+
+The legacy PowerShell SDK documentation covers the details of creating help for PowerShell
+cmdlets packaged into modules. However, PowerShell does not provide any tools for creating the
+XML-based help. The SDK documentation explains the structure of MAML help, but leaves you the
+task of creating the complex, and deeply nested, MAML content by hand.
+
+This is where the [PlatyPS][] module can help.
+
+## What is PlatyPS?
+
+PlatyPS is an [open-source][platyps-repo] tool that started as a _hackathon_ project to make the
+creation and maintenance of MAML easier. PlatyPS documents the syntax of parameter sets and the
+individual parameters for each cmdlet in your module. PlatyPS creates structured [Markdown][]
+files that contain the syntax information. It can't create descriptions or provide examples.
+
+PlatyPS creates placeholders for you to fill in descriptions and examples. After adding the required
+information, PlatyPS compiles the Markdown files into MAML files. PowerShell's help system also
+allows for plain-text conceptual help files (about topics). PlatyPS has a cmdlet to create a
+structured Markdown template for a new _about_ file, but these `about_*.help.txt` files must be
+maintained manually.
+
+You can include the MAML and Text help files with your module. You can also use PlatyPS to compile
+the help files into a CAB package that can be hosted for updateable help.
+
+## Get started using PlatyPS
+
+First you must install PlatyPS from the PowerShell Gallery.
+
+```powershell
+Install-Module platyps -Force
+Import-Module platyps
+```
+
+The following flowchart outlines the process for creating or updating PowerShell reference content.
+
+:::image type="content" source="./media/create-help-using-platyps/cmdlet-ref-flow-v2.png" alt-text="The workflow for creating XML-based help using PlatyPS":::
+
+##  Create Markdown content for a PowerShell module
+
+1. Import your new module into the session. Repeat this step for each module you need to document.
+
+   Run the following command to import your modules:
+
+   ```powershell
+   Import-Module <your module name>
+   ```
+
+1. Use PlatyPS to generate Markdown files for your module landing page and all associated cmdlets
```

Comment:
Do you think we should define what a Module Landing page is?

Reply:
I can expand on this.